### ISSUE TITLE
{"file":"not found"} error

### ISSUE THREAD

**Issue by rstacruz** [pwmckenna/node-travis-encrypt#8]
```
travis-encrypt -r rstacruz/abc KEY=VAL
# KEY
[object Object]
```

Inspecting `[object Object]`, it's actually `err` as reported by `encrypt()` which translates to `{"file":"not found"}`.

That is:

```js
> require('travis-encrypt')('a/b', 'KEY=VAL', undefined, undefined, console.log)
...
{ file: 'not found' }
```

Using 1.0.2 from npm.

**Comment by burnnat** [comment]
I'm experiencing this too, on Windows 7 x86 (not sure if this is OS-dependent). Anybody know of a workaround?

**Comment by rstacruz** [comment]
I'm getting this on OSX 10.10 + Node.js 0.10, so I can confirm this isn't something that would be OS-dependent :)

**Comment by pwmckenna** [comment]
i'll fix the [object Object] display issue...as for the actual error, what repo slug are you actually using? is it private?

**Comment by burnnat** [comment]
Ok, so I think I've figured this out, at least for my situation: I had mistakenly enabled the wrong repo in Travis, so when I switched on the correct repository in the Travis site and re-ran it completed successfully.

Maybe a more enlightening error message might be in order for this case?

**Comment by pwmckenna** [comment]
it would be fairly straight forward to at least add error handling for this specific case. travis' api doesn't generally provide the most helpful messages though, so being helpful in the most common cases is probably the best we can do.

as a side note, I didn't really know if people were using this out in the wild. it probably deserves at least a small test suite now that it has at least 3 users :)

**Comment by pwmckenna** [comment]
@burnnat @rstacruz thanks for the input, just released 1.1.1 that does an additional check if the key lookup fails, and tells you if it can't find the repo specified.

> repository pwmckenna/node-travis-encrypts not found

the error formatting issue should also be resolved (though I had to remove the red colorizing or the error message). I'm going to add you both as contributors, so if you see additional areas for improvement, feel free.

**Comment by rstacruz** [comment]
I ran into this as part of https://medium.com/@nthgergo/publishing-gh-pages-with-travis-ci-53a8270e87db by the way!
